samples/drivers/rtc/ds1307/README.rst

```diff
@@ -0,0 +1,33 @@
+DS1307: RTC Module
+#######################################################

```

Comment:
In RST generally you make the heading specifier just as long as your actual heading.